lib/elixir/lib/list.ex

```diff
@@ -2,14 +2,17 @@ defmodule List do
   @moduledoc """
   Functions that work on (linked) lists.
   
-  Please note that most of the functions provided for lists--which implement 
-  the `Enumerable` protocol--are found in `Enum`.
+  Many of the functions provided for lists, which implement 
+  the `Enumerable` protocol, are found in the `Enum` module.
   
-  Additionally, the following functions for lists are found in `Kernel`:
+  Additionally, the following functions and operators for lists are
+  found in `Kernel`:
   
+    * `Kernel.++/2`
+    * `Kernel.--/2`
```

Comment:
`++/2` and `--/2`  should work with latest exdoc version

Reply:
I prefer the `Kernel` prefix though from outer modules for clarity.

Reply:
So in/2 should use Kernel